### PR TITLE
Upgraded Cellranger related docker images

### PR DESCRIPTION
**Pull request by HarrisonFisher**

### CODE REVIEW

File: docker/cellranger-atac/2.1.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64-2.4.23.zip" -o "awscliv2.zip" && \
+    unzip awscliv2.zip && \
+    ./aws/install && \
+    rm awscliv2.zip
+
+RUN ln -s /usr/bin/python3 /usr/bin/python
+
+RUN python -m pip install --upgrade pip && \
+    python -m pip install pandas==1.3.5 && \
```
Comment (yihming) [inline docker/cellranger-atac/2.1.0/Dockerfile:18]: Upgrade to pandas `1.4.2`.

File: docker/cellranger-atac/2.1.0/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    python -m pip install packaging==21.3 && \
+    python -m pip install stratocumulus==0.1.5
```
Comment (yihming) [inline docker/cellranger-atac/2.1.0/Dockerfile:20]: Upgrade to stratocumulus `0.1.7`.

File: docker/cellranger-atac/2.1.0/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,32 @@
+FROM --platform=linux/amd64 debian:bullseye-slim
+SHELL ["/bin/bash", "-c"]
+
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y alien unzip build-essential dpkg-dev curl gnupg procps python3 python3-pip && \
+    echo "deb https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && \
+    curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add - && \
+    apt-get update && apt-get install -y google-cloud-cli=375.0.0-0
```
Comment (yihming) [inline docker/cellranger-atac/2.1.0/Dockerfile:8]: Upgrade to google-cloud-cli `389.0.0-0`.

File: docker/cellranger-atac/2.1.0/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64-2.4.23.zip" -o "awscliv2.zip" && \
```
Comment (yihming) [inline docker/cellranger-atac/2.1.0/Dockerfile:10]: Upgrade to aws-cli `2.7.7`.

**Review by yihming — CHANGES_REQUESTED** [review]

**Review by HarrisonFisher — COMMENTED** [review]
Resolved the changes

**Review by yihming — APPROVED** [review]

**Pull request merged**